mpd/mpd.go

```diff
@@ -639,23 +640,24 @@ func (period *Period) AddNewAdaptationSetVideoWithID(id string, mimeType string,
 // Create a new Adaptation Set for Subtitle Assets.
 // mimeType - MIME Type (i.e. text/vtt).
 // lang - Language (i.e. en).
```

Comment:
Either remove this comment or add the new label parameter. Same for the other functions that were changed.

Reply:
Added the new label parameter on the comments for all the functions on the latest commit